mautic/mautic
# Feature install plugins command

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? | Y
| Related user documentation PR URL | TODO
| Related developer documentation PR URL | /
| Issues addressed (#s or URLs) | /
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This PR refactor the plugin actions like install and update so they could be reused in a `app/console mautic:plugins:install` command. Command included.

#### Steps to test this PR:
1. Try to hit the Install button on the Plugins page. A message should show up.
2. Try to install a new plugin or re-install a plugin you have already there.
3. Trigger the command above and you should get the same message in the CLI as you get in the UI.
